pkg/apis/policy/v1alpha1/override_types.go

```diff
@@ -100,6 +102,31 @@ type Overriders struct {
 	// ArgsOverrider represents the rules dedicated to handling container args
 	// +optional
 	ArgsOverrider []CommandArgsOverrider `json:"argsOverrider,omitempty"`
+
+	// LabelsOverrider represents the rules dedicated to handling workload labels
+	// +optional
+	LabelsOverrider []CommandKeyValuePairOverrider `json:"labelsOverrider,omitempty"`
+
+	// AnnotationsOverrider represents the rules dedicated to handling workload annotations
+	// +optional
+	AnnotationsOverrider []CommandKeyValuePairOverrider `json:"annotationsOverrider,omitempty"`
+}
+
+// CommandKeyValuePairOverrider represents the rules dedicated to handling workload labels/annotations
+type CommandKeyValuePairOverrider struct {
+
+	// Operator represents the operator which will apply on the workload.
+	// +kubebuilder:validation:Enum=add;remove;replace
+	// +required
+	Operator OverriderOperator `json:"operator"`
+
+	// Value to be applied to annotations/labels of workload.
+	// Items in Value which will be appended after annotations/labels when Operator is 'add'.
+	// Items in Value which match in annotations/labels will be deleted when Operator is 'remove'.
+	// Items in Value which match in annotations/labels will be replaced when Operator is 'replace'.
+	// If Value is empty, then the annotations/labels will remain the same.
+	// +optional
+	Value map[string]string `json:"value,omitempty"`
```

Comment:
I'm thinking if there is a better name for `Value`, but at least we should use value`s` here.

> If Value is empty, then the annotations/labels will remain the same.

Should we set the field with `required`? because it might look weird if `value` is empty.

Reply:
> I'm thinking if there is a better name for Value, but at least we should use values here.

I want to keep the same style as other `Overrider` to reduce user learning costs


https://github.com/karmada-io/karmada/blob/b5119c391a0a02bc8443016d754d4b741bad0638/pkg/apis/policy/v1alpha1/override_types.go#L147

https://github.com/karmada-io/karmada/blob/b5119c391a0a02bc8443016d754d4b741bad0638/pkg/apis/policy/v1alpha1/override_types.go#L202

https://github.com/karmada-io/karmada/blob/b5119c391a0a02bc8443016d754d4b741bad0638/pkg/apis/policy/v1alpha1/override_types.go#L176

Reply:
@RainbowMango what do you think?

Reply:
Yeah, I'm sold. That makes sense.

Reply:
But I think this field should be `required`. (Ignore the previous overriders for this PR. If we decide to change the previous ones, we might need to start `v1alpha2`.)

Reply:
> But I think this field should be required.

+1 

> we might need to start v1alpha2

This is a very important decision, which may require the community to participate in the discussion.